components/sidebar/add_channel_dropdown.tsx

```diff
@@ -105,19 +110,32 @@ class AddChannelDropdown extends React.PureComponent<Props, State> {
             </Tooltip>
         );
 
+        let tutorialTip = null;
+        if (this.props.showTutorialTip) {
+            tutorialTip = (
+                <AddChannelTutorialTip
+                    townSquareDisplayName={this.props.townSquareDisplayName}
+                    offTopicDisplayName={this.props.offTopicDisplayName}
+                />
+            );
+        }
+
         return (
             <MenuWrapper className='AddChannelDropdown'>
                 <OverlayTrigger
                     delayShow={500}
                     placement='top'
                     overlay={tooltip}
                 >
-                    <button
-                        className='AddChannelDropdown_dropdownButton'
-                        aria-label={intl.formatMessage({id: 'sidebar_left.add_channel_dropdown.dropdownAriaLabel', defaultMessage: 'Add Channel Dropdown'})}
-                    >
-                        <i className='icon-plus'/>
-                    </button>
+                    <>
```

Comment:
nit: why the fragment tag?

Reply:
OverlayTrigger only allows a single react child. I got build errors without the fragment.